### PR TITLE
[DD-899] adapt uuid and some bugfixes

### PR DESCRIPTION
**Pull request by chrode**

### CODE REVIEW

File: yo/app/scripts/directives/data-config-csv.js [hunk 1]
```diff
@@ -93,31 +96,27 @@ angular.module('dmpApp')
         function extractFromResource(dataResource) {
             if (dataResource.configurations) {
 
-                var latestConfig = loDash.max(dataResource.configurations, 'id');
+                var latestConfig = dataResource.configurations[0];
```
Comment (knutwalker) [inline yo/app/scripts/directives/data-config-csv.js:100]: This doesn't seem too correct, we probably have to address this at some point. IIRC you've already mentioned this at the last meeting. Can we have a TODO or an issue?

**Comment by knutwalker** [comment]
ping @zazi 

Overall, there might be some conflicts, since the backend uses the pattern "entity-uuid", e.g. "attribute-de305d54-75b4-431b-adb2-eb6b9e546013". I think it's only to reduce the chance of collisions.
Although it might work with mixed styles, I think we should agree on one.
I give my :+1: for unqualified uuids, that is, just "de305d54-75b4-431b-adb2-eb6b9e546013".
In addition to this [SO answer](http://stackoverflow.com/a/4306474/2996265), UUID4 has an entropy of 122 bits which gives about 5e36 possiblities, It is somewhat estimated, that the overall internet traffic is about one zetabyte. Assuming a collision at .5, we can index 250 million million times the complete internet, byte for byte, before we get a collision. I call that unlikely :)

**Comment by knutwalker** [comment]
Otherwise, LGTM.

**Comment by boferri** [comment]
@knutwalker we can go for plain UUID4 right now. it shouldn't even matter, if we mix up both styles (i.e. with and without prefixes). at the end everything will be stored as a simple string (so no real uuid it required at all ;) ).

**Pull request merged**